service/internal/auth/casbin_test.go

```diff
@@ -7,10 +7,13 @@ import (
 	"testing"
 
 	"github.com/casbin/casbin/v2/model"
+	gormadapter "github.com/casbin/gorm-adapter/v3"
```

Comment:
Could we use a SQL adapter? We already use `pgx` and we have examples of using the same driver https://github.com/opentdf/platform/blob/4f6825e3370f0617f443659ddeec8b1b0f751b15/service/pkg/db/db.go#L15